cgosqlite/cgosqlite.go

```diff
@@ -90,10 +90,30 @@ type DB struct {
 	declTypes map[string]string
 }
 
+// cStmt is a wrapper around an sqlite3 *sqlite3_stmt. Except it's stored as
```

Comment:
Nit: "stmt, except"

Reply:
Kinda subjective. I prefer keeping the first sentence short and to the point. But I reworded the second sentence in https://github.com/tailscale/sqlite/pull/62